### PR TITLE
Add recipe for DynaMate

### PR DESCRIPTION
**Pull request by DarkmatterVale**
This PR adds a recipe for DynaMate.

@diversys I used PE so all of the recipe should implement proper tabbing.

### CODE REVIEW

**Comment by korli** [comment]
Please squash the 2 commits.

**Comment by pulkomandy** [comment]
The copyright is not correct (my fault, I did not fix github default LICENSE file). I fixed it now, can you adjust the recipe?

**Comment by diversys** [comment]
Also please squash your commits

**Comment by DarkmatterVale** [comment]
Commits have been squashed and the Copyright updated.

GCI task: https://codein.withgoogle.com/dashboard/task-instances/5942099685933056/

File: games-puzzle/dynamate/dynamate-1.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Dynamate
+
+	cp -r DynaMate/* $appsDir/Dynamate/
```
Comment (diversys) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: mkdir -p $appsDir
cp -r DynaMate $appsDir

This should also work and is shorter

File: games-puzzle/dynamate/dynamate-1.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+1995 Jonas Norberg, Signe Norberg and Bjorn Kalzen"
+LICENSE="MIT"
+SOURCE_URI="git+https://github.com/HaikuArchives/Dynamate.git#e870f3c3026739831b749318d58826b0e4d874d7"
+REVISION="1"
+
+ARCHITECTURES="x86 ?x86_gcc2 ?x86_64"
+
+PROVIDES="
+	dynamate = $portVersion
+	app:dynamate = $portVersion
```
Comment (diversys) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: should be app:DynaMate = $portVersion

File: games-puzzle/dynamate/dynamate-1.0.0.recipe (hunk 1 above)
Comment (DarkmatterVale) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: I was testing and this did not work on my machine

File: games-puzzle/dynamate/dynamate-1.0.0.recipe (hunk 2 above)
Comment (DarkmatterVale) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: Should I also change "dynamate = $portVersion" to "DynaMate = $portVersion"?

Comment (diversys) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: No, the first line is the package name which is always lowercased.

Comment (DarkmatterVale) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: Ok

File: games-puzzle/dynamate/dynamate-1.0.0.recipe (hunk 1 above)
Comment (DarkmatterVale) [inline games-puzzle/dynamate/dynamate-1.0.0.recipe]: Got it to work

**Comment by DarkmatterVale** [comment]
Does anyone have any more comments?

**Comment by pulkomandy** [comment]
Yes: the final package should also include the levels, documentation, and level editor (currently called LevelWorld).

**Comment by DarkmatterVale** [comment]
@pulkomandy Should I also include EditWorld?

**Comment by pulkomandy** [comment]
No, EditWorld is an older editor and I don't think it is useful (or even usable) with current versions of the game. I should probably delete it from the repository someday, but I fear of the code being lost.

**Comment by pulkomandy** [comment]
(the recipe is still missing the documentation and levels. did you put these in a separate recipe and forgot to git add it before pushing?)

**Comment by DarkmatterVale** [comment]
No, they are included in the Dynamate folder which contains the main application.

**Pull request merged**